pages/home.py

```diff
@@ -64,6 +65,8 @@
                                 id=ElementsIDs.CHART_CONTAINER.value,
                             ),
                             dmc.Text(id=ElementsIDs.note_model.value),
+                            dcc.Location(id=ElementsIDs.URL.value, refresh=False),
+                            dcc.Store(id="url-initialized", storage_type="memory"),
```

Comment:
add "url-initialized" to ElementsIDs and do not use a string here

Reply:
Fixed in `d0bca97` commit

Reply:
> add "url-initialized" to ElementsIDs and do not use a string here

fixed in `d0bca97` commit